test/helpers/utils.js

```diff
@@ -40,18 +40,27 @@ exports.checkOutput = function (f, argv, cb) {
 
     f()
   } else {
-    result = f()
+    try {
+      result = f()
+    } finally {
+      reset()
```

Comment:
Yeah, figured as much… do you feel like this is something you’d like to see addressed or are you fine with it this way?